src/additional-tachyons/_animation.css

```diff
@@ -8,3 +11,44 @@
 @media(--breakpoint-large) {
   .ta-l { transition: all 0.3s; }
 }
+
+/*****************
+ * Spin
+ ****************/
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
```

Comment:
That's a really interesting though - I'd never thought about it before. It _does_ seem like that should be true, but I don't believe it actually behaves that way. Every spinner I've looked at uses 360deg, and I've never noticed a stutter. How interesting!

Check out this example (not my code, but uses the same keyframes): http://jsfiddle.net/raving/e2tt2mao/